index.html

```diff
@@ -147,6 +161,7 @@ <h1>I’m Abhishek <br>
             </div>
         </div>
     <div class="describe">
+        <div class="abt-self">
```

Comment:
- [ ] Kindly, be sure to inspect your design with a resolution of 1440px width. Another thing is that the boxes containing the languages are not the same as Figma's. They should have the same size and their content should not be centered. 👍 

![Captura de Pantalla 2022-09-30 a la(s) 4 49 19 p  m](https://user-images.githubusercontent.com/86022690/193361949-60cd4c79-b2a0-4b19-873f-d17a717840b0.png)

I can still see the same issues that the previous reviewer pointed out. Please fix them 🤓

Reply:
i am not seeing this problem on my screen

Reply:
I cloned your project locally and used other online methods to host your code. It shows that one of the things that the previous reviewers pointed out is still present and that is the grey background I pointed out

Reply:
there is no gap untill you go on 125% and if i am decreasing margin there than the contact will overlap on about section